.github/workflows/autobuild.yaml

```diff
@@ -122,26 +122,44 @@ jobs:
         "-DCMAKE_CXX_FLAGS_RELEASE=${{ matrix.config.cflags }}"
         ${{ matrix.config.cmake }}
     - name: Ninja
-      run: ninja -C ${{ env.CLANGD_DIR }} clangd
-    - name: Archive
+      run: ninja -C ${{ env.CLANGD_DIR }} clangd clangd-indexer
+    - name: Archive clangd
       run: >
         7z a clangd.zip
         ${{ env.CLANGD_DIR }}/LICENSE.TXT
-        ${{ env.CLANGD_DIR }}/bin/clangd*
+        ${{ env.CLANGD_DIR }}/bin/clangd
         ${{ env.CLANGD_DIR }}/lib/clang
-    - name: Upload artifact
+    - name: Archive indexing-tools
+      run: >
+        7z a indexing-tools.zip
+        ${{ env.CLANGD_DIR }}/LICENSE.TXT
+        ${{ env.CLANGD_DIR }}/bin/clangd-indexer
+    - name: Upload clangd artifact
       uses: actions/upload-artifact@v1
       with:
         name: ${{ matrix.config.name }}
         path: clangd.zip
-    - name: Upload asset
+    - name: Upload indexing tools artifact
```

Comment:
sorry if i am being dense, but why do we need both uploading as an artifact and as a release asset?

is it ever possible for artifact to be generated but release asset to be missing? (i knows this already existed in the old code, but maybe we should drop that rather than adding more :D)

Reply:
Yeah that was more about consistency. I don't know if there is a reason to have both artifacts & release assets at the same time here because IIUC artifacts are mostly used to share files between jobs (as in the case of `periodic` and `autobuild` ones) and I don't know how artifacts could be useful otherwise.

@sam-mccall is there anything you rely on I'm missing that comes with having the artifact uploaded in the building job?

Reply:
I removed this for now, let me know if it is actually a required step @sam-mccall.